includes/signature/class-draft-cavage-signature.php

```diff
@@ -132,7 +124,7 @@ public function verify( array $headers, $body = null ) {
 	 * @return string The digest.
 	 */
 	public function generate_digest( $body ) {
-		return 'sha256=' . \base64_encode( \hash( 'sha256', $body, true ) );
+		return 'SHA-256=' . \base64_encode( \hash( 'sha256', $body, true ) );
```

Comment:
why this change? doesn't it break signatures then? or are they currently broken?